net-libs/libblkmaker/libblkmaker-0.5.3.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A library implementing getblocktemplate"
+DESCRIPTION="libblkmaker provides an implementation of getblocktemplate, the\
+ current Bitcoin mining protocol."
+HOMEPAGE="https://github.com/bitcoin/libblkmaker"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libblkmaker/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="549193640d4cc7022725754833d9d8a7639c41e1a7d2c5f2588be65857cc0970"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [[ "$targetArchitecture" == "x86_gcc2" && -z "$secondaryArchSuffix" ]]; then
+    PATCHES="${portVersionedName}.patchset"
+fi
+
+PROVIDES="
+    libblkmaker${secondaryArchSuffix} = $portVersion
+    lib:libblkmaker_0.1${secondaryArchSuffix} = $portVersion
+    lib:libblkmaker_jansson_0.1${secondaryArchSuffix} = $portVersion
+    devel:libblkmaker_0.1${secondaryArchSuffix} = $portVersion
+    devel:libblkmaker_jansson_0.1${secondaryArchSuffix} = $portVersion
+"
+
+REQUIRES="
+    haiku${secondaryArchSuffix}
+    libbase58${secondaryArchSuffix}
+    jansson${secondaryArchSuffix}
+"
+
+BUILD_REQUIRES="
+    haiku${secondaryArchSuffix}_devel
+    pkgconfig
+    cmd:gcc${secondaryArchSuffix}
+    cmd:make
+    cmd:autoreconf
+    cmd:aclocal
+    cmd:libtool
```

Comment:
You should move all the cmd: lines from the BUILD_REQUIRES to a BUILD_PREREQUIRES bloc (below BUILD_REQUIRES).
And the `cmd:libtool` should be replaced by `cmd:libtoolize$secondaryArchSuffix`